ui/appui-react/src/appui-react/stagepanels/StagePanelDef.tsx

```diff
@@ -102,7 +110,6 @@ export class StagePanelDef extends WidgetHost {
       const panel = frontstageDef.nineZoneState.panels[side];
       if (panel.size === this._size)
         return;
-      size = panel.size;
```

Comment:
The new size of the panel should be reported to the def, so this line should remain. (otherwise, if I say: `StagePanelDef.size = 10;` and the min value is 20, the panel will be set to 20, but the inner memory will be kept as if it was 10, which is not true.

Reply:
Good catch!